### PR TITLE
Add imageContainerStyle prop for styling the image

### PR DESCRIPTION
**Pull request by srameshr**
custom styles for the image

### CODE REVIEW

File: components/Column.js [hunk 1]
```diff
@@ -82,7 +82,8 @@ export default class Column extends Component {
     return bricks.map((brick, index) => {
```
Comment (brh55) [inline components/Column.js]: @srameshr Can you add imageContainerStyle to the Columns.PropTypes? Here as well as the `Masonry` component.

Comment (srameshr) [reply components/Column.js]: Sure! On it.

Comment (srameshr) [reply components/Column.js]: Done

**Comment by brh55** [comment]
Will bump this package to v0.3.1 to make it immediately accessible to the public. The optimization is a relatively big change, so things will hit v0.4.0 relatively soon.

**Pull request merged**